project_tests.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-
-tox
-. .tox/py3/bin/activate
-pip install coveralls
-COVERALLS_REPO_TOKEN=$(cat /etc/coveralls/tokens/elife-crossref-xml-generation) coveralls
-
+set -e
+. mkvenv.sh
+source venv/bin/activate
+pip install pytest coverage 
```

Comment:
you should add `wheel` and `pip` in there as well as an `--upgrade`, so:

    pip install pip wheel pytest coverage --upgrade

`pytest` and `coverage` should probably live in `requirements.txt`

Reply:
Thanks, I wasn't sure how important those are - adding!

Reply:
wheel lets you install wheels and pip will use wheels if it can. Older versions of pip may not use wheels or fail to find the correct one etc. The default pip in use also can't be guaranteed as it may be the first time it was upgraded from the stock system pip.